integration/sawtooth_integration/docker/two_fams.yaml

```diff
@@ -0,0 +1,96 @@
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+version: "2.1"
+
+services:
+
+  tp_config:
+    image: sawtooth-tp_config:$ISOLATION_ID
+    volumes:
+      - ../../../:/project/sawtooth-core
+    expose:
+      - 40000
+    depends_on:
+      - validator
+    entrypoint: ./bin/tp_config -vv tcp://validator:40000
+    stop_signal: SIGKILL
+
+  tp_intkey:
+    image: sawtooth-tp_intkey_python:$ISOLATION_ID
+    volumes:
+      - ../../../:/project/sawtooth-core
+    expose:
+      - 40000
+    depends_on:
+      - validator
+    entrypoint: ./bin/tp_intkey_python -vv tcp://validator:40000
+    stop_signal: SIGKILL
+
+  tp_xo:
+    image: sawtooth-tp_xo_python:$ISOLATION_ID
+    volumes:
+      - ../../../:/project/sawtooth-core
+    expose:
+      - 40000
+    depends_on:
+      - validator
+    entrypoint: ./bin/tp_xo_python -vv tcp://validator:40000
+    stop_signal: SIGKILL
+
+  validator:
+    image: sawtooth-validator:$ISOLATION_ID
+    volumes:
+      - ../../../:/project/sawtooth-core
+    expose:
+      - 40000
+    # start the validator with an empty genesis batch
+    entrypoint: "bash -c \"\
+        ./bin/sawtooth admin keygen && \
+        ./bin/sawtooth admin genesis && \
+        ./bin/validator -vv\""
+    stop_signal: SIGKILL
+
+  rest_api:
+    image: sawtooth-rest_api:$ISOLATION_ID
+    volumes:
+      - ../../../:/project/sawtooth-core
+    expose:
+      - 40000
+      - 8080
+    depends_on:
+      - validator
+    entrypoint: ./bin/rest_api --stream-url tcp://validator:40000
+    stop_signal: SIGKILL
+
+  integration_test:
+    image: sawtooth-test:$ISOLATION_ID
+    volumes:
+      - ../../../:/project/sawtooth-core
+    expose:
+      - 8080
+    working_dir: /project/sawtooth-core/integration/sawtooth_integration/tests
+    entrypoint: python3 -m http.server
```

Comment:
This is an accident, that this initially got in to the intkey and xo smokes, so shouldn't be here.